website/docs/r/repository_collaborators.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
A link to this document is missing in the `github.erb` file. Please add a reference so this doc is discoverable!

Reply:
done.